packages/vectors/src/point-at.ts

```diff
@@ -0,0 +1,38 @@
+import { Vec, maddN } from ".";
+import { maddN2, maddN3 } from "./maddn";
+
+/**
+ * Calculates the point laying on ray at given distance
+ *
+ *
+ * @param out -
+ * @param rayDir -
+ * @param rayOrigin -
+ * @param dist -
+ */
+export function pointAtOnRay(
```

Comment:
@nkint mille grazie, but a couple of comments before we can merge:

1. I think `rayOrigin ` and `rayDir` should be swapped, not only to be more uniform w/ the order used in other functions (e.g. [`intersectRayPlane`](https://github.com/thi-ng/umbrella/blob/develop/packages/geom-isec/src/ray-plane.ts)), but also because how people usually think of a ray as a start point plus a direction vector, not dir + point. That order is also consistent with pretty much every other literature, project... :)
2. Can you please refactor them all as arrow functions? See: https://github.com/thi-ng/umbrella/blob/develop/CONTRIBUTING.md#arrow-functions-preferred 😺 
3. I'd say we have to decide on one preposition here in the name, either `pointAtRay` or `pointOnRay`. I'd opt for the latter or else `pointAtDistance`. Thoughts?

Reply:
ok, thanks